### PR TITLE
add new favicons

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/codeigniter4projects/website/issues/303

### CODE REVIEW

File: public/favicons/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" width="266.667" height="266.667"><svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="266.667" height="266.667" viewBox="0 0 200 200"><path d="M100.7 15.5c1.1 11.6-5.8 22-27.8 42-23.5 21.4-35.1 36.6-41 53.8-10.1 29.4 1.2 57.5 28.4 70.7 9.7 4.7 12.6 5.2 7.5 1.2-10.9-8.2-15.2-20.4-11.9-33 1.4-5.2 8.2-15.3 12.9-19.2 2-1.6 3.9-3 4.3-3 .3 0 .4 2.2.1 4.8-.3 4.1 0 5.2 2.2 7.9 7.4 8.8 20.6 4.9 20.6-6.2 0-4.4-.9-6.1-8.7-16.4-7.3-9.6-7.9-18-2-26.8 4-5.8 5.2-6.2 5.3-1.9.1 5.7 4.7 11 15.8 18.3 21.3 13.9 23.7 15.6 29.4 21.8 6.7 7.1 9.8 12.7 11.2 20.7 2.3 12.4-3.8 25.1-16 33.1-5.2 3.4-2.7 2.9 6.6-1.3 17.9-8.1 32.2-23.1 37-38.9 2.1-6.9 1.8-20.6-.6-28.3-3.7-12-13.6-24.4-26.3-32.8-3.2-2.2-6.2-4-6.6-4-.4 0-1.5 1.6-2.4 3.5-3.2 6.7-12.4 9.6-18.7 5.8-7.2-4.2-9-14.2-4.9-27.4 1.7-5.3 2.3-9.8 2.3-15.9 0-7.5-.3-9.3-3.2-15.3-1.8-3.7-5.7-9.3-8.6-12.4l-5.4-5.8.5 5z" fill="#dd4814"></path></svg><style>@media (prefers-color-scheme: light) { :root { filter: none; } }
+</style></svg>
```
Comment (MGatner) [inline public/favicons/favicon.svg:2]: Do the line breaks have to be awkward? Can the first line be split up more, or make all this into one line?

**Review by MGatner — CHANGES_REQUESTED** [review]
The favicon sections are completely duplicated across the two layouts and should probably be extracted into their own view file. That doesn't need to be done in this PR, but please respond about SVG file format for line breaks.

File: public/favicons/favicon.svg (hunk 1 above)
Comment (Arhell) [reply public/favicons/favicon.svg]: Done

**Review by MGatner — APPROVED** [review]
Looks good!

**Pull request merged**